test/services/licences/fetch-agreements.service.test.js

```diff
@@ -8,46 +8,62 @@ const { describe, it, beforeEach } = exports.lab = Lab.script()
 const { expect } = Code
 
 // Test helpers
-const DatabaseSupport = require('../../support/database.js')
 const FinancialAgreementHelper = require('../../support/helpers/financial-agreement.helper.js')
 const LicenceAgreementHelper = require('../../support/helpers/licence-agreement.helper.js')
 
 // Thing under test
-const FetchAgreementsService =
-  require('../../../app/services/licences/fetch-agreements.service.js')
+const FetchAgreementsService = require('../../../app/services/licences/fetch-agreements.service.js')
 
 describe('Fetch Agreements service', () => {
-  const licenceRef = '01/12/34/1000'
-
-  beforeEach(async () => {
-    await DatabaseSupport.clean()
-  })
+  let licenceAgreement
+  let financialAgreement
 
   describe('when the licence has agreements data', () => {
     beforeEach(async () => {
```

Comment:
Having this as a `beforeEach` will cause it to create 2 records, one for each test. You only really need one Financial Agreement record for these tests so you can change this to a `before`. You'll also need to add `before` to line 7.
```suggestion
    before(async () => {
```